views/thank_you.erb

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+     <link rel="stylesheet" href="//aui-cdn.atlassian.com/aui-adg/5.9.12/css/aui.min.css" media="all">
+    <script src="<%= $fqdn %>/atlassian-connect/all.js"></script>
+  </head>
+   <body>
+      <div class="ac-content">
+        <div class="aui-message aui-message-info">
+              Thank you for installing GitHub App Plugin for JIRA. Please return to a JIRA ticket to create a branch.
```

Comment:
When setting up an App, make sure to set `Setup URL (optional)` back to your instance. This will redirect back to your application with the `installation_id` - see https://github.com/osowskit/atlassian-github-app-addon/issues/9